src/lib.rs

```diff
@@ -287,8 +290,8 @@ pub trait BuildArgs: Downcast {
             .filter_map(|x| match x {
                 Ok(x) => Some(x),
                 Err(err) => {
-                    eprintln!(
-                        "WARNING: could not walk into directory `{}`: {}",
+                    warn!(
```

Comment:
```suggestion
                    log::warn!(
```

Use the log crate directly so we don't have to maintain the list of imports up there.